tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/model/Root.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.ei.tools.mule2ballerina.model;
+
+import org.wso2.ei.tools.mule2ballerina.visitor.Visitable;
+import org.wso2.ei.tools.mule2ballerina.visitor.Visitor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Queue;
+
+/**
+ * {@code Root} This is the root of the internal MTree
+ */
+public class Root extends BaseObject implements Visitable {
+
+    private List<GlobalConfiguration> globalConfigurations;
+    private Queue<Flow> flowList;
+    private Map<String, GlobalConfiguration> configMap;
+
+    public Root() {
+        flowList = new LinkedList<Flow>();
+        globalConfigurations = new ArrayList<GlobalConfiguration>();
+        configMap = new HashMap<String, GlobalConfiguration>();
+    }
+
+    public List<GlobalConfiguration> getGlobalConfigurations() {
+        return globalConfigurations;
+    }
+
+    public Queue<Flow> getFlowList() {
+        return flowList;
+    }
+
+    public void addGlobalConfiguration(GlobalConfiguration globalConfiguration) {
```

Comment:
Let's add java doc for functions

Reply:
Since getters and setters are self-explanatory, they were not added.